_summaries/lang2018pointpillars_2.md

```diff
@@ -1,19 +1,51 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: lang2018pointpillars
 # Please fill out info below
-author: # Your GitHub id
+author: aabayomi
 score: # How did you like this paper 0(dislike) to 10(love)
 ---
-
+<!-- 
 TODO: Summarize the paper:
 * What is the core idea?
 * How is it realized (technically)?
 * How well does the paper perform?
-* What interesting variants are explored?
+* What interesting variants are explored? -->
+
+# Core Idea
+
+The authors proposes as much faster approach by which encoder that learns point cloud representations by stacking learned features as vertical tensors. 
+
+# Main Contributions
+
+![](lang2018pointpillars_2a.png)
+
+* Representation of the 3-D lidar point cloud inputs as pseudo-images
+
+* 2-D convolutional backbone processes the images
+
+* Single-shot detectors regresses to 3D boxes
+
+# Loss function 
+
+The loss formulation combines the sotfmax classification loss, focal loss on object classification  and localization loss.This is optimized using Adam of learning rate of $2 * 10^−4$ with a decay rate 0.8 at every 15 epoch.
+
+![](lang2018pointpillars_2b.png)
+
+
+# Performance
+
+On KITTI datasets PointPillars  outperforms fusion based methods on cars and cyclists my respect to mean average precision (mAP) compared to lidar-only methods.
+
+![](lang2018pointpillars_2c.png)
+
+PointPillar is faster compared to SOTA on BEV
+
+![](lang2018pointpillars_2d.png)
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+
+* PointPillars encoder is computationally efficient and performant on object detection. 
```

Comment:
You can add a point on lidar point clouds. One point on encoder of how it learns point cloud in a vertical column (pillars) which results in 2D conv layers and hence faster computation.

Reply:
Another TLDR point could be about its performance compared with previous models or it's generalizability that any backbone can be used.